game/scripts/vscripts/Sections/Chapter2/SectionOpening.ts

```diff
@@ -85,32 +82,47 @@ const onStart = (complete: () => void) => {
             ),
 
             // Talking about moonwells
-            tg.withHighlights(tg.seq([
-                tg.audioDialog(LocalizationKey.Script_2_Opening_2, LocalizationKey.Script_2_Opening_2, context => context[CustomNpcKeys.SunsFanMudGolem]),
-                tg.audioDialog(LocalizationKey.Script_2_Opening_3, LocalizationKey.Script_2_Opening_3, context => context[CustomNpcKeys.SlacksMudGolem]),
-            ]), {
-                type: "circle",
-                units: Entities.FindAllByClassname("npc_dota_filler").concat(Entities.FindAllByClassname("npc_dota_effigy_statue")) as CDOTA_BaseNPC[],
-                radius: 150,
-                attach: false,
-            }
-            ),
+            tg.fork([
+                tg.panCameraLinear(_ => getPlayerCameraLocation(), _ => Entities.FindAllByClassnameWithin("npc_dota_filler", moveNextToBarracksLocation, 1000)[0].GetAbsOrigin(), 1),
+                tg.setCameraTarget(undefined),
```

Comment:
This line should be unnecessary as panCameraLinear already sets the camera target to its dummy